plugin.video.fatstone/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.fatstone"
+       name="Fatstone.TV"
+       version="1.0.0"
+       provider-name="trippel-m">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="script.module.requests" version="1.0.4"/>
```

Comment:
If you intend to submit this for Jarvis and versions later, please set xbmc.python version="2.24.0"
Please see http://kodi.wiki/view/Addon.xml#Dependency_versions for other version values, if you want something other than Jarvis